config.toml

```diff
@@ -1,4 +1,4 @@
-baseURL = "/tech-department/"
+baseURL = "/"
 languageCode = "en-US"
```

Comment:
Please fix this. This change will break the website

Reply:
sorted